experimental/generation/generator/packages/library/test/forms/unittest_keyPhrase_with_pattern.schema

```diff
@@ -0,0 +1,19 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "properties": {
+        "KeyphraseProperty": {
+            "type": "string",
+            "$entities": [
+                "keyPhrase"
+            ],
```

Comment:
Add keyPhrase.schema

Reply:
You should be able to $ref the keyphrase schema in place of type/$entities.

---
In reply to: [513838723](https://github.com/microsoft/BotBuilder-Samples/pull/2854#discussion_r513838723) [](ancestors = 513838723)